package.json

```
@@ -35,7 +35,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "sass": "1.57.1",
-    "vanilla-framework": "4.0.0-alpha.3"
+    "vanilla-framework": "4.0.0-rc.1"
```

As 4.0.0 has been released in the meantime, I'd suggest switching to it.

```suggestion
    "vanilla-framework": "4.0.0"
```